src/util/io.c

```diff
@@ -30,11 +30,65 @@
 #include <string.h>
 #include <unistd.h>
 
-#include "sockets.h"
-#include "tcti.h"
-#define LOGMODULE tcti
+#include "io.h"
+#define LOGMODULE io
```

Comment:
Having to many log modules may start to be problematic. Now we get the trance for what is being sent after we set TSS2_LOG=tcti+TRACE. After that we will need to have TSS2_LOG=tcti+TRACE,io+TRACE, which isn't too bad, but adding more may take some time to setup. We should keep the number of log modules to minimum.

Reply:
I was thinking about this when I made the change but hadn't considered the state space explosion. I've never used anything more than `TSS2_LOG=all+TRACE` but that's not to say the issue doesn't exist. How about we just use a single logging module for everything in the util library? That way we have one for each library and one additional one for the utility code?

Reply:
That would work for now since we will only have the io and the log itself in utils, but with more code utils this will defeat the purpose of logs filtering. It should rather be a logical separation.

Reply:
I'll flip this back to `tcti` for the log module since that's the only place where this code is used. We can deal with this issue when it becomes an issue. Will fixup and push.

Reply:
sounds good